gstreamer-mpegts-sys/build.rs

```diff
@@ -16,22 +16,8 @@ fn main() {
 fn find() -> Result<(), Error> {
     let package_name = "gstreamer-mpegts-1.0";
     let shared_libs = ["gstmpegts-1.0"];
-    let version = if cfg!(feature = "v1_14") {
-        "1.14"
-    } else if cfg!(feature = "v1_12") {
+    let version = {
         "1.12"
-    } else if cfg!(feature = "v1_10") {
-        "1.10"
-    } else if cfg!(feature = "v1_8") {
-        "1.8"
-    } else if cfg!(feature = "v1_6") {
-        "1.6"
-    } else if cfg!(feature = "v1_4") {
-        "1.4"
-    } else if cfg!(feature = "v1_2") {
-        "1.2"
-    } else {
-        "1.0"
```

Comment:
This all should stay

Reply:
As above. I am guessing it is directly related to the above and the Meson / Autotools issue.

Reply:
Same as above, simply revert